Submissions/PratyushPareek/task2.py

```diff
@@ -0,0 +1,16 @@
+import shutil
+import os
+
+ 
+for folders, subfolders ,files in os.walk("..\\...\\Monty_Python_Class\\Task_2\\My_Files\\'):
```

Comment:
Use os.path.join()

Reply:
@Lokesh2703  done 
I've also made a separate branch and a separate PR for task 3